scripts/upload_image.py

```diff
@@ -98,10 +100,29 @@ def image_from_clipboard() -> Image:
 
 
 class Uploader:
-    def __init__(self, pngcrush: bool):
+    def __init__(self, pngcrush: bool, auto_accept: bool):
         gcs = storage.Client("rerun-open")
         self.bucket = gcs.bucket("rerun-static-img")
         self.run_pngcrush = pngcrush
+        self.auto_accept = auto_accept
+
+    def _check_aspect_ratio(self, image: Path | Image):
+        if isinstance(image, Path):
+            image = PIL.Image.open(image)
+        aspect_ratio = image.width / image.height
+
+        if not (self.auto_accept or (ASPECT_RATIO_RANGE[0] < aspect_ratio < ASPECT_RATIO_RANGE[1])):
```

Comment:
```suggestion
        if self.auto_accept or not (ASPECT_RATIO_RANGE[0] < aspect_ratio < ASPECT_RATIO_RANGE[1]):
```

Reply:
And yes, the logic looks wrong

Reply:
ok, just ignore me

Reply:
I guess it would be more readable:

``` suggestion
aspect_ok = ASPECT_RATIO_RANGE[0] < aspect_ratio < ASPECT_RATIO_RANGE[1]
if not aspect_ok and not self.auto_accept:
```

Reply:
That looks best indeed 👍🏻